nixos/tests/nginx-sandbox.nix

```diff
@@ -10,6 +10,7 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     nixpkgs.overlays = [
       (self: super: {
         nginx-lua = super.nginx.override {
+          pcre = pkgs.pcre;
```

Comment:
can we fix this in the package, somehow?
people needing to override `pcre` isn't great IMO.

Reply:
lua module is not built with pcre2 support. I was unable to build the latest version of the lua module:
```
...
nginx> checking for execvpe ... found
nginx> ./configure: /nix/store/j3imz21r5hzlpgw599jn1msgfx24h2f4-lua/config: line 514: can't create /nix/store/j3imz21r5hzlpgw599jn1msgfx24h2f4-lua/src/ngx_http_lua_autoconf.h: Permission denied
...
nginx>  src/stream/ngx_stream_ssl_preread_module.c
nginx> make[1]: *** No rule to make target '/nix/store/j3imz21r5hzlpgw599jn1msgfx24h2f4-lua/src/ngx_http_lua_autoconf.h', needed by 'objs/addon/2ysp5ichpccf4lv1wp2qcwz0bmm840f1-rtmp/ngx_rtmp.o'.  Stop.
nginx> make[1]: *** Waiting for unfinished jobs....
nginx> make[1]: Leaving directory '/build/nginx-1.22.0'
nginx> make: *** [Makefile:10: build] Error 2
error: builder for '/nix/store/69dnphwla7k6688px8mgbsd24nrb9883-nginx-1.22.0.drv' failed with exit code 2;

```
I don't know how to fix the `ngx_http_lua_autoconf.h` file write error.

Reply:
There is a possibility that the latest version does not support pcre2.

Reply:
our lua module is a bit out of date (we have 0.10.15, newest seems 0.10.21), but there's also https://github.com/openresty/lua-nginx-module/issues/1984

Reply:
As a temporary solution, I replaced pcre with pcre2.
Or do you have another solution?